src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -316,7 +315,6 @@ class SidebarLinks extends React.Component {
                     optionMode={this.props.priorityMode === CONST.PRIORITY_MODE.GSD ? 'compact' : 'default'}
                     onLayout={App.setSidebarLoaded}
                 />
-                <KeyboardSpacer />
```

Comment:
@rushatgabhane by removing `<KeyboardSpacer />` it solves one more issue do I link another issue here?

Reply:
on to it, I'm preparing the root cause for that will link soon :)